internal/schemautil/schemautil_test.go

```diff
@@ -112,7 +111,7 @@ func TestResourceDataSetAddForceNew(t *testing.T) {
 	// Missing force_new field
 	d := mocks.NewMockResourceData(t)
 	err := ResourceDataSet(d, dto, s)
-	require.Error(t, err, fmt.Errorf("%w: %q", errMissingForceNew, "force_new"))
+	require.ErrorIs(t, err, errMissingForceNew)
```

Comment:
The error message points out that the missing field is `force_new`, and not any other. Please keep the message check.

Reply:
Added additional check `require.ErrorContains(t, err, "force_new")`